### PR TITLE
Added Eloquent DataCollector

### LINKED ISSUE #47: Add Eloquent data collector

### PR DESCRIPTION
**Pull request by wouterj**
/fixes #47 

sf-eloquent-profiler

### CODE REVIEW

**Comment by narutimateum** [comment]
what does this do??

**Comment by wouterj** [comment]
This adds a data collector. This way, all executed queries during a request, the time it took to execute them and the used connections will be displayed in the web debug toolbar.

File: src/WouterJEloquentBundle.php [hunk 1]
```diff
@@ -35,6 +37,18 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new ObserverPass(), PassConfig::TYPE_BEFORE_OPTIMIZATION, 80);
     }
 
+    public function registerCommands(Application $application)
```
Comment (theofidry) [inline src/WouterJEloquentBundle.php]: is that change related?

File: src/Twig/SqlFormatterExtension.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace WouterJ\EloquentBundle\Twig;
+
+class SqlFormatterExtension extends \Twig_Extension
+{
+    public function getFilters()
```
Comment (theofidry) [inline src/Twig/SqlFormatterExtension.php]: missing `inheritdoc` blocs

File: src/Twig/SqlFormatterExtension.php [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace WouterJ\EloquentBundle\Twig;
+
+class SqlFormatterExtension extends \Twig_Extension
```
Comment (theofidry) [inline src/Twig/SqlFormatterExtension.php]: maybe mark this class as `@private`? or do you want to apply the BC policy on that one as well?

File: src/Twig/SqlFormatterExtension.php [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<?php
+
```
Comment (theofidry) [inline src/Twig/SqlFormatterExtension.php]: missing license

File: src/DependencyInjection/WouterJEloquentExtension.php [hunk 5]
```diff
@@ -40,6 +41,18 @@ public function load(array $configs, ContainerBuilder $container)
         $this->loadCapsule($config, $container, $loader);
         $this->loadEloquent($config, $container, $loader);
         $this->loadFacades($config, $container, $loader);
+        $this->loadDataCollector($container, $loader);
+    }
+
+    protected function loadDataCollector(ContainerBuilder $container, Loader\XmlFileLoader $loader)
```
Comment (theofidry) [inline src/DependencyInjection/WouterJEloquentExtension.php]: missing `inheritdoc`
